### DISCUSSION TITLE
project pull request error

### DISCUSSION THREAD

**Discussion by Manikandanramaraj2008** [community/community#134239] [Pull Requests]
### Select Topic Area

Show & Tell

### Body

I am not able to do pull request for my project, below is the errorm message pls guide me

>>> C:/Program Files/Git/bin/git.exe commit -F C:/Users/maniv/AppData/Local/Temp/RtmpGW1Uu0/git-commit-message-2c646c971f84.txt --amend 

--gpg-sign
gpg: skipped "maniv <maniramkr2006@gmail.com>": No secret key
gpg: signing failed: No secret key
error: gpg failed to sign the data
fatal: failed to write commit object

**Comment by aslbekDev07** [comment]
hello

**Comment by BhathiyaPrasad** [comment]
try to setup ssh key

**Reply by Manikandanramaraj2008** [reply]
Thanks... i will do it and update you if it works

**Reply by BhathiyaPrasad** [reply]
sure, please mark as answered if it works
